sros2/sros2/keystore/_keystore.py

```diff
@@ -61,61 +59,53 @@ def create_keystore(keystore_path):
-    else:
-        print('found CA key and cert, not creating new ones!')
-
-    # create governance file
-    gov_path = os.path.join(keystore_path, _KS_ENCLAVES, 'governance.xml')
-    if not os.path.isfile(gov_path):
-        print('creating governance file: %s' % gov_path)
+        _utilities.create_symlink(
+            src=pathlib.Path('ca.cert.pem'), dst=keystore_permissions_ca_cert_path)
+        _utilities.create_symlink(
+            src=pathlib.Path('ca.key.pem'), dst=keystore_permissions_ca_key_path)
+        _utilities.create_symlink(
+            src=pathlib.Path('ca.cert.pem'), dst=keystore_identity_ca_cert_path)
+        _utilities.create_symlink(
+            src=pathlib.Path('ca.key.pem'), dst=keystore_identity_ca_key_path)
+
+    # Create governance file if it doesn't already exist
+    gov_path = keystore_path.joinpath(_KS_ENCLAVES, 'governance.xml')
+    if not gov_path.is_file():
         _create_governance_file(gov_path, _utilities.domain_id())
-    else:
-        print('found governance file, not creating a new one!')
 
-    # sign governance file
-    signed_gov_path = os.path.join(keystore_path, _KS_ENCLAVES, 'governance.p7s')
-    if not os.path.isfile(signed_gov_path):
-        print('creating signed governance file: %s' % signed_gov_path)
+    # Sign governance file if it hasn't already been signed
+    signed_gov_path = keystore_path.joinpath(_KS_ENCLAVES, 'governance.p7s')
+    if not signed_gov_path.is_file():
         _utilities.create_smime_signed_file(
             keystore_permissions_ca_cert_path,
             keystore_permissions_ca_key_path,
             gov_path,
             signed_gov_path)
-    else:
-        print('found signed governance file, not creating a new one!')
-
-    print('all done! enjoy your keystore in %s' % keystore_path)
-    print('cheers!')
-    return True
 
 
-def is_valid_keystore(path):
+def is_valid_keystore(path: pathlib.Path) -> bool:
     return (
-        os.path.isfile(os.path.join(path, _KS_PUBLIC, 'permissions_ca.cert.pem')) and
-        os.path.isfile(os.path.join(path, _KS_PUBLIC, 'identity_ca.cert.pem')) and
-        os.path.isfile(os.path.join(path, _KS_PRIVATE, 'permissions_ca.key.pem')) and
-        os.path.isfile(os.path.join(path, _KS_PRIVATE, 'identity_ca.key.pem')) and
-        os.path.isfile(os.path.join(path, _KS_ENCLAVES, 'governance.p7s'))
+        path.joinpath(_KS_PUBLIC, 'permissions_ca.cert.pem').is_file() and
+        path.joinpath(_KS_PUBLIC, 'identity_ca.cert.pem').is_file() and
+        path.joinpath(_KS_PRIVATE, 'permissions_ca.key.pem').is_file() and
+        path.joinpath(_KS_PRIVATE, 'identity_ca.key.pem').is_file() and
+        path.joinpath(_KS_ENCLAVES, 'governance.p7s').is_file()
     )
 
 
-def get_keystore_enclaves_dir(keystore_path: str) -> str:
-    return os.path.join(keystore_path, _KS_ENCLAVES)
+def get_keystore_enclaves_dir(keystore_path: pathlib.Path) -> pathlib.Path:
+    return keystore_path.joinpath(_KS_ENCLAVES)
```

Comment:
```suggestion
    return keystore_path / _KS_ENCLAVES
```

Reply:
Throughout the PR joinpath can be replaced with the / operator, which simply appends the path/string on the right